packages/common/src/search/_internal/hubSearchItems.ts

```diff
@@ -18,7 +20,9 @@ export async function hubSearchItems(
   query: IQuery,
   options: IHubSearchOptions
 ): Promise<IHubSearchResponse<IHubSearchResult>> {
-  return options.aggFields?.length
-    ? searchOgcAggregations(query, options)
-    : searchOgcItems(query, options);
+  if (options.aggFields?.length) {
+    return searchOgcAggregations(query, options);
+  }
+  const url = `${getOgcCollectionUrl(query, options)}/items`;
+  return searchOgcItems(url, query, options as ISearchOgcItemsOptions);
```

Comment:
Note that `ISearchOgcItemsOptions` is just a subset of `IHubSearchOptions[]`, so the cast should be fine here.